Dockerfile

```diff
@@ -17,10 +17,10 @@ RUN GOEXPERIMENT=boringcrypto CGO_ENABLED=1 GOOS=linux go build -o /go/bin/ratel
 GOEXPERIMENT=boringcrypto CGO_ENABLED=1 GOOS=linux go build -o /go/bin/ratelimit_config_check -ldflags="-w -s" -v github.com/replicon/ratelimit/src/config_check_cmd
 
 
-FROM 434423891815.dkr.ecr.us-east-1.amazonaws.com/machine-images/fips-base:m-11320-amazon-linux-2 AS final
+FROM 434423891815.dkr.ecr.us-east-1.amazonaws.com/machine-images/fips-base:m-11878-amazon-linux-2 AS final
 
 RUN yum install -y python3 && \
-  pip3 install ipaddress awscli && \
+  pip3 install ipaddress && \
```

Comment:
Removing `awscli` here because the fips-base image comes with aws-cli v2 as seen [here](https://github.com/replicon/machine-images/blob/master/fips-base/amazon-linux-2/Dockerfile#L8)